### PR TITLE
Copy update - Privacy notice - Online purchase

### PR DESCRIPTION
**Pull request by bethcollins92**
## Done

- Copy update - Online purchase page

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/legal/data-privacy/online-purchase
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Check page matches [copy doc](https://docs.google.com/document/d/1ZSNnMiLJl6SGSV0D_cr6MJAwpkf6xzbKYJdTL9MxLHc/edit#heading=h.75k0ordixf1t.)

## Issue / Card

Fixes [#3202](https://github.com/canonical-web-and-design/web-squad/issues/3202)

### CODE REVIEW

File: templates/legal/data-privacy/online-purchase.md [hunk 1]
```diff
@@ -24,35 +24,104 @@ When you purchase products from us online, we ask you for your name, address, co
...
+
+When purchasing a product from us online, our legal basis for processing your personal information is that it is necessary for our legitimate interests in conducting our business and meeting the requirements of you, our current and prospective customers.
+
+We may contact you from time to time with information about products and services that we offer (see [Marketing Communications to Businesses](https://ubuntu.com/legal/data-privacy/partner-portal#mkt-comms)). Our legal basis for using your personal information to contact you is that it is necessary for our legitimate interest in keeping you informed of the products and services that we offer. You can opt out of marketing communications as set out below.
+
+#### *Your consent*
+If you purchase a product from us online, you will have the option to provide your consent to us sending you information about the products and services that we offer (see [Email Marketing Communications](https://ubuntu.com/legal/data-privacy/partner-portal#mkt-comms)). You can withdraw your consent at any time.
```
Comment (pmahnke) [inline templates/legal/data-privacy/online-purchase.md:53]: please use relative links
```suggestion
If you purchase a product from us online, you will have the option to provide your consent to us sending you information about the products and services that we offer (see [Email Marketing Communications](/legal/data-privacy/partner-portal#mkt-comms)). You can withdraw your consent at any time.
```

File: templates/legal/data-privacy/online-purchase.md [hunk 2]
```diff
@@ -24,35 +24,104 @@ When you purchase products from us online, we ask you for your name, address, co
...
+
+Your credit card details are passed to a third-party payment processor who retains credit card details.
 
 Our third-party payment processor is a company called Stripe Payments Europe, Limited, who we use to process and confirm your payments. We share with them your name, address and credit card information only for payment purposes. We do not retain credit card data but our payment service providers do. For your safety, we have in place agreement between us to protect, safeguard your information and prohibit any sharing for any reasons whatsoever.
 
 We also use a third-party supplier called Shopify Inc. to provision your order and we share with them your name, address and telephone number in order to complete the delivery.
 
+## Our legal basis to use your Personal information
+
+#### *Our legitimate business interests*
```
Comment (steverydz) [inline templates/legal/data-privacy/online-purchase.md:45]: These should be h3 without emphasis

File: templates/legal/data-privacy/online-purchase.md [hunk 3]
```diff
@@ -24,35 +24,104 @@ When you purchase products from us online, we ask you for your name, address, co
...
+Our legal basis for using your personal information is that it is necessary for our legitimate interest in providing the purchased product in accordance with the terms and conditions between us and the business you are associated with.
+
+When purchasing a product from us online, our legal basis for processing your personal information is that it is necessary for our legitimate interests in conducting our business and meeting the requirements of you, our current and prospective customers.
+
+We may contact you from time to time with information about products and services that we offer (see [Marketing Communications to Businesses](https://ubuntu.com/legal/data-privacy/partner-portal#mkt-comms)). Our legal basis for using your personal information to contact you is that it is necessary for our legitimate interest in keeping you informed of the products and services that we offer. You can opt out of marketing communications as set out below.
```
Comment (pmahnke) [inline templates/legal/data-privacy/online-purchase.md:50]: relative links
```suggestion
We may contact you from time to time with information about products and services that we offer (see [Marketing Communications to Businesses](/legal/data-privacy/partner-portal#mkt-comms)). Our legal basis for using your personal information to contact you is that it is necessary for our legitimate interest in keeping you informed of the products and services that we offer. You can opt out of marketing communications as set out below.
```

File: templates/legal/data-privacy/online-purchase.md (hunk 2 above)
Comment (steverydz) [inline templates/legal/data-privacy/online-purchase.md:45]:
```suggestion
### Our legitimate business interests
```

File: templates/legal/data-privacy/online-purchase.md [hunk 4]
```diff
@@ -24,35 +24,104 @@ When you purchase products from us online, we ask you for your name, address, co
...
+
+#### *Your consent*
```
Comment (steverydz) [inline templates/legal/data-privacy/online-purchase.md:52]:
```suggestion
### Your consent
```

**Review by steverydz — COMMENTED** [review]
A few comments in the code

File: templates/legal/data-privacy/online-purchase.md (hunk 2 above)
Comment (bethcollins92) [reply templates/legal/data-privacy/online-purchase.md:45]: @steverydz the "Your consent" (line 52) is written the same as "Our legitimate business interests" (line 45) in the copy doc, so should I change it to h3 no emphasis also?

**Review by steverydz — APPROVED** [review]
LGTM :+1:

**Pull request merged**